test/c/end2end/end2end_test.cc

```diff
@@ -0,0 +1,370 @@
+/*
```

Comment:
This will eventually be the case. Right now C tests are compiled with a C++ server.